packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1091,13 +1101,8 @@ export default class Meeting extends StatelessSparkPlugin {
         deviceUrl: this.deviceUrl,
         reason
       })
-      .then((response) => {
+      .then(() => {
         this.ringing(STOP, false);
-        Events.emit(EVENTS.DESTROY_MEETING_1_1, {
-          meeting: this,
-          response,
-          type: EVENTS.DESTROY_MEETING_1_1
-        });
```

Comment:
Are we no longer emitting an event when a meeting is ended by the server?

Reply:
The check for  maxParticipants = 3 indicates if its a 1:1 call . For other meeting maxParticipants is 25 to 200

We dont emit destroy because the call could be still active if you have joined on other device and left here. We wait for the server event in locusInfo to decide that

Reply:
Why is maxParticipants 3 for a 1:1? What is the other participant?

Reply:
normally its 2 but when guest user joins its 3 so the server sends the maxParticipants as 3 so participant length will be below that . maxParticipants length is always less then 3

Reply:
@mattnorris  i commented here